databricks-sdk-java/src/main/java/com/databricks/sdk/core/DatabricksCliCredentialsProvider.java

```diff
@@ -5,21 +5,21 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class BricksCliCredentialsProvider implements CredentialsProvider {
+public class DatabricksCliCredentialsProvider implements CredentialsProvider {
 
-  private static final Logger LOG = LoggerFactory.getLogger(BricksCliCredentialsProvider.class);
+  private static final Logger LOG = LoggerFactory.getLogger(DatabricksCliCredentialsProvider.class);
 
-  public static final String BRICKS_CLI = "bricks-cli";
+  public static final String DATABRICKS_CLI = "databricks-cli";
 
   @Override
   public String authType() {
-    return BRICKS_CLI;
+    return DATABRICKS_CLI;
   }
 
-  private CliTokenSource getBricksCliTokenSource(DatabricksConfig config) {
-    String cliPath = config.getBricksCliPath();
+  private CliTokenSource getDatabricksCliTokenSource(DatabricksConfig config) {
+    String cliPath = config.getDatabricksCliPath();
     if (cliPath == null) {
-      cliPath = "bricks";
+      cliPath = "databricks";
     }
```

Comment:
Please add logic that would distinguish legacy (pre-`v0.200`) and new (`v0.200`+) versions. See the reference implementation in https://github.com/databricks/databricks-sdk-py/blob/main/databricks/sdk/core.py#L301-L327